azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -506,6 +512,7 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str]):
         kusto_cluster = kcsb.data_source
 
         # Create a session object for connection pooling
+        # self._session is only used using sync access

```

Comment:
why? can't use session in async flow?

Reply:
Async operations are wrapped by aiohttp.ClientSession which is way way faster:
![image](https://user-images.githubusercontent.com/6255867/74101235-56888880-4b40-11ea-8f32-518aaddf42dd.png)
https://julien.danjou.info/python-and-fast-http-clients/